File: Erlang/start-code/startcode.erl
-module(startcode).
-export([start/0]).

start() ->
    io:format("Willkommen zu meinem Erlang-Programm!~n"),
    io:format("Bitte geben Sie Ihren Namen ein: "),
    Name = io:get_line(""),
    NameTrimmed = string:strip(Name),
    io:format("Hallo, ~s! Schön, dass Sie hier sind.~n", [NameTrimmed]),
    io:format("Das Programm wurde beendet.~n").

File: Nim/start-code/startcode.nim
# Willkommen zu meinem Nim-Programm

import strutils

proc startCode() =
  echo "Willkommen zu meinem Nim-Programm!"
  echo "Bitte geben Sie Ihren Namen ein: "
  let name = readLine(stdin)
  echo "Hallo, ", name.strip(), "! Schön, dass Sie hier sind."
  echo "Das Programm wurde beendet."

startCode()
